index.md

```diff
@@ -37,20 +37,20 @@ title: Sean Lo's Resume
+* Spearheaded the development and maintenance of a Python Appium Client/Server pytest testing framework for PS4, PS5, Windows, and Web that was utilized by 1000+ of developers and testers, leading to increased developer productivity and 20% less service downtime
+* Designed and implemented CI/CD a release pipeline utilizing Jenkins, Bash, and Groovy, enabling automated semantic releases for 30+ teams delivering PlayStation React Native-based applications
+* Supported a crucial PlayStation Network tool in JavaScript with Node.js and MongoDB, processing 100,000+ of calls weekly across API, web, and CLI for development and testing
+* Proactively led refactoring effort of multiple Python-based internal tools leveraged by 20+ teams daily for development, testing, and nightly regression jobs, resulting in a 44% decrease in job processing times and 70% increase in developer productivity
 * Crafted a Python-based Telemetry app with MongoDB and Grafana, enabling shareholder insights and improving KPI monitoring and decision-making
-* Developed and led training workshops and tutorials to 200+ QA and developers each session on the testing framework and numerous other tools my team created
+* Developed and led training workshops and KT sessions to 200+ QA and developers each session on the testing framework and numerous other tools I created
 * Provided mentorship to team members, assisting them in navigating their career paths while sharing my expertise
 
 ### **Software Development Engineer in Test** - *PlayStation (Sony)* 
 
 <span> San Francisco, CA <span class="small-right">Mar. 2021 - Nov. 2021</span></span>
 
-* Assisted in the successful launch of PlayStation Direct's expansion to Europe for 10+ countries reaching 10,000,000+  monthly visitors 
-* Developed Selenium automation strategy for PlayStation Direct's checkout system and drove implementation of automated end-to-end tests suite ensuring 100% test coverage with a focus on stability and maintainability
+* Played a crucial role in the successful launch of PlayStation Direct's expansion to Europe for 10+ countries reaching 10,000,000+  monthly visitors 
+* Developed Selenium automation strategy for PlayStation Direct's checkout system and drove implementation of automated end-to-end tests suite ensuring 100% test coverage, improving stability and maintainability
 * Designed, developed, and enhanced internal JavaScript Selenium WebDriver libraries for interacting with web interfaces on the PlayStation websites, ensuring robust and efficient testing processes
 * Implemented and integrated CI/CD strategies using Groovy and Jenkins to improve testing and development efficiency and effectiveness
 * Worked in collaboration with the QA Lead, developers, project manager and various stakeholder teams to ensure the continuous delivery of product releases
```

Comment:
kind of ambiguous. release frequency or stability improved? Team achieve anything through these efforts you can mention?